### DISCUSSION TITLE
Decrease number of outgoing reports

### DISCUSSION THREAD

**Discussion by dzikiBOB** [Koenkk/zigbee2mqtt#20191] [Devices]
Gentlemen, is it possible to change the reporting conditions for some devices?
I have a CO2 sensor
https://www.zigbee2mqtt.io/devices/TS0601_air_quality_sensor.html#tuya-ts0601_air_quality_sensor
and Energy Meter
https://www.zigbee2mqtt.io/devices/SPM01-D2TZ.html#yagusmart-spm01-d2tz

Both devices report with alarming frequency every 2-3 seconds, e.g. when the measured voltage value or the measured CO2 value changes.
This causes huge and unnecessary traffic at the z2m>MQTT>openhab interface.
I looked in the Reporting of these devices tab, but none of the parameters visible there match...

I think that limiting the report to sending once every 60 seconds or when changing by 2 units would improve working conditions and cut off voltage changes such as 238>237>238>237>238>237...

**Comment by felipebraz** [comment]
Did you find a solution?
